db/migrate/20221118125948_add_track_to_submissions.rb

```diff
@@ -0,0 +1,16 @@
+class AddTrackToSubmissions < ActiveRecord::Migration[7.0]
+  def change
+    unless Rails.env.production?
+      add_column :submissions, :track_id, :smallint, null: true
```

Comment:
No foreign key, right?

Reply:
Correct. We can't because we've used `smallint` here anyway